src/tqdm_publisher/_subscriber.py

```diff
@@ -1,10 +1,10 @@
-from typing import Any, Dict, Iterable
+from typing import Any, Dict, Iterable, Optional
 
 from ._publisher import TQDMProgressPublisher
 
 
 class TQDMProgressSubscriber(TQDMProgressPublisher):
-    def __init__(self, iterable: Iterable[Any], on_progress_update: callable, **tqdm_kwargs):
+    def __init__(self, on_progress_update: callable, iterable: Optional[Iterable[Any]] = None, **tqdm_kwargs):
```

Comment:
To not break back compatability based on argument ordering going forward, maybe best to enforce keyword only usage here?

```suggestion
    def __init__(self, *, on_progress_update: callable, iterable: Optional[Iterable[Any]] = None, **tqdm_kwargs):
```

Reply:
Actually, the hope would be to mirror standard `tqdm` usage as much as possible. In that case, we should enforce that `on_progress_update` is a required kwarg while allowing the user to pass anything else as they'd like.

Reply:
@garrettmflynn OK, edited suggestion for that

Reply:
@CodyCBakerPhD How does this look? https://github.com/catalystneuro/tqdm_publisher/pull/65/commits/3c85e514245fb3642b99266654b179e18a42fff9

Reply:
Fine by me